packages/api/src/external/carequality/document/process-outbound-document-retrieval-resps.ts

```diff
@@ -29,6 +29,33 @@ export async function processOutboundDocumentRetrievalResps({
     let issuesWithGateway = 0;
     let successDocsCount = 0;
 
+    if (results.length === 0) {
+      const msg = `No results received to process.`;
+
+      log(`${msg}`);
```

Comment:
nit and NABD, but could leave the log/capture code in one single block so we can focus on logic when we read the code (understanding the empty lines approach to make code standout, logs and capture are "infra"/piping)